pkgs/applications/networking/p2p/zeronet/default.nix

```diff
@@ -12,24 +13,24 @@ python2Packages.buildPythonApplication rec {
   };
 
   propagatedBuildInputs = with python2Packages; [ msgpack gevent ];
-
-  format = "other";
-
   buildPhase = "${python2Packages.python.interpreter} -O -m compileall .";
 
   installPhase = ''
     mkdir -p $out/share
     cp -r plugins src tools *.py $out/share/
   '';
 
-  # Wrap the main executable and set the log and data dir to something out of
-  # the store
+  # Wrap the main executable and set the log and data dir to something out of the store
   postFixup = ''
     makeWrapper "$out/share/zeronet.py" "$out/bin/zeronet" \
-        --set PYTHONPATH "$PYTHONPATH" \
-        --set PATH ${python2Packages.python}/bin \
-        --add-flags "--log_dir \$HOME/.local/share/zeronet/logs" \
-        --add-flags "--data_dir \$HOME/.local/share/zeronet"
+      --set PYTHONPATH "$PYTHONPATH" \
+      --set PATH ${python2Packages.python}/bin \
+      --add-flags "--log_dir \$HOME/.local/share/zeronet/logs" \
+      --add-flags "--data_dir \$HOME/.local/share/zeronet"
+
+    makeWrapper "$out/share/zeronet.py" "$out/bin/zeronet_service" \
+      --set PYTHONPATH "$PYTHONPATH" \
+      --set PATH ${python2Packages.python}/bin
```

Comment:
I think you can just use the first wrapper. If you pass a second set of `--log_dir` and `--data_dir`, it will override the first.

Reply:
That's already defined in the conf this service generates.
Better for ease-of-use and less possible problems.